packages/components/ovh-at-internet/src/ovh-at-internet.ts

```diff
@@ -297,13 +307,25 @@ export default class OvhAtInternet extends OvhAtInternetConfig {
   trackClick(data: LegacyTrackingData): void {
     if (this.canTrack()) {
       const pageTrackingData = getPageTrackingData(data);
-      const tracking = {
+      let tracking: any = {
         ...this.getGenericTrackingData(data),
         click: pageTrackingData.page,
         click_chapter1: pageTrackingData.page_chapter1,
         click_chapter2: pageTrackingData.page_chapter2,
         click_chapter3: pageTrackingData.page_chapter3,
       };
+      if (data?.page) {
+        const pageTrackingData2 = getPageTrackingData(data?.page);
+        tracking = {
+          ...tracking,
+          page: pageTrackingData2.page,
+          page_chapter1: pageTrackingData2.page_chapter1,
+          page_chapter2: pageTrackingData2.page_chapter2,
+          page_chapter3: pageTrackingData2.page_chapter3,
+          page_complete: data?.page?.name,
+        };
+      }
```

Comment:
**question:** Can you please explain me this? Why do we need to execute `getPageTrackingData` on `data` as well as `data.page`?

Reply:
sure i will update it